plugins/modules/proxysql_query_rules.py

```diff
@@ -355,23 +368,29 @@ def __init__(self, module):
                             "match_digest",
                             "match_pattern",
                             "negate_match_pattern",
+                            "re_modifiers",
                             "flagOUT",
                             "replace_pattern",
                             "destination_hostgroup",
                             "cache_ttl",
-                            "cache_empty_result",
```

Comment:
`cache_empty_result` is not needed to handle in this PL...can be reverted.

Reply:
Ok, blame me.

`next_query_flagIN` and `next_query_flagIN` both exists also in 1.4.15.

Reply:
Ah alright, I'll leave it then. But if next_query_flagIN already existed this version constraint is not required then I guess